### ISSUE TITLE
aria-hidden with focusable content fails the wrong succescriterium

### ISSUE THREAD

**Issue by danistr** [auto-wcag/auto-wcag#448]
```
<div aria-hidden="true">
	<a href="/" >Link</a>
</div>
```

According to the rule this is currently a violation under success criterion 4.1.2. (Name, Role, Value) but should be a violiation on success criterion 1.3.1 (Info and Relationships). For example a link. Visually there is a link visible but for screenreader users this link is no longer perceivable and will not be read.
It can't fail under success criterion 4.1.2. because there is a name and role present and a aria-hidden attribute doesn't remove this.

**Comment by annethyme** [comment]
Handled by https://github.com/auto-wcag/auto-wcag/pull/449
